build.gradle

```diff
@@ -73,10 +85,15 @@ dependencies {
     compile 'org.slf4j:slf4j-api:1.7.25'
     compile 'ch.qos.logback:logback-core:1.2.3'
     compile 'ch.qos.logback:logback-classic:1.2.3'
-    compile 'com.datastax.cassandra:cassandra-driver-core:3.6+'
+
+    compile 'com.datastax.cassandra:cassandra-driver-core:3.8+'
     compile group: 'com.google.inject', name: 'guice', version: '4.2.2'
     compile group: 'org.apache.commons', name: 'commons-configuration2', version: '2.4'
 
+    compile(group: 'com.google.guava', name: 'guava', version: '28.2-jre') {
```

Comment:
We don't need this. The guava version update in the buildscript is sufficient.

Reply:
We still need it.  Guava isn't JDK 11 compatible till version 27, and the buildscript block only affects the build tooling, not the artifacts.  I've verified this locally.

Reply:
For reference on the Guava side: https://github.com/google/guava/issues/3272

Reply:
Here's the buildscan of the dependency tree if I remove the explicit Guava block:

![image](https://user-images.githubusercontent.com/337872/75592821-2d209580-5a38-11ea-939e-8c0c1a9eecbe.png)

when I build it with the forced guava version:

![image](https://user-images.githubusercontent.com/337872/75592844-3a3d8480-5a38-11ea-9da8-ea20c010a4e7.png)

Reply:
I understand. I would like to take a conservative approach in upgrading guava. Here's my concern -

```
+--- com.datastax.cassandra:cassandra-driver-core:3.6+ -> 3.6.0
|    +--- io.netty:netty-handler:4.0.56.Final -> 4.1.30.Final (*)
|    +--- com.google.guava:guava:19.0 -> 28.2-jre
|    |    +--- com.google.guava:failureaccess:1.0.1
|    |    +--- com.google.guava:listenablefuture:9999.0-empty-to-avoid-conflict-with-guava
```

This might actually break the driver. So, ideally we should go to a driver that supports Guava 28. Right now v3.8 of the driver seems to be breaking the integration tests so we cannot upgrade that. Let's address this as a follow up ticket?

Reply:
Fair enough, I'll update my branch.